### ISSUE TITLE
Support of serde v1.x.x

### ISSUE THREAD

**Issue by arazabishov** [abonander/anterofit#11]
Hi @abonander, 

Thanks for the great work on library. Are there any plans to support first stable release of serde?

**Comment by abonander** [comment]
Yeah, it's in the works along with #6. I'm still working on https://github.com/abonander/multipart-async though.

**Comment by arazabishov** [comment]
Nice! Do you accept third party contributions?

**Comment by abonander** [comment]
On which project?

**Comment by arazabishov** [comment]
Anterofit.

**Comment by abonander** [comment]
Yeah, if you want an upgrade of Serde now feel free to try it. I was thinking I'd lump all these changes into one release but it doesn't have to be that way.
